boards/CMakeLists.txt

```diff
@@ -430,10 +428,10 @@ function(create_arm_binary_or_tests_for_board
     git_hash_code_generation(${GIT_HASH_HEADER})
 
     if ("${PLATFORM}" STREQUAL "x86")
-        compile_googletest_executables(
+        compile_googletest_executable(
```

Comment:
Renamed to a singular "executable", because while, we only compile one executable per board now. (Just making it clear that the renaming was intentional)